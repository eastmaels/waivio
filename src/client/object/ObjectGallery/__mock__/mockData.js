export const wObj = {
  id: 'xjn-chicken-katsu',
  name: 'galleryAlbum',
  body: 'Photos',
  items: [
    {
      weight: 2,
      locale: 'en-US',
      _id: '5ccf4fd2d0555b20e13bc695',
      creator: 'diningguide',
      author: 'zxc43',
      permlink: 'diningguide-v4dbssu79i',
      name: 'galleryItem',
      body: 'https://ipfs.busy.org/ipfs/Qmejn8Mnectr5siH6DVHtUFwExpFMTYU9zAY6eMLDzrmzB',
      id: 'xjn-chicken-katsu',
      active_votes: [
        {
          _id: '5ccf4fd6d0555b20e13bc95c',
          voter: 'diningguide',
          weight: 1,
        },
      ],
    },
    {
      weight: 2,
      locale: 'en-US',
      _id: '5ccf5015d0555b20e13c704b',
      creator: 'diningguide',
      author: 'xcv47',
      permlink: 'diningguide-a7hp8jyjvuh',
      name: 'galleryItem',
      body: 'https://ipfs.busy.org/ipfs/QmbsqEx8iRLusNQqgrBi5NSUdmcNoeNx2gHEcUAs5HG274',
      id: 'xjn-chicken-katsu',
      active_votes: [
        {
          _id: '5ccf5019d0555b20e13c794e',
          voter: 'diningguide',
          weight: 1,
        },
      ],
    },
  ],
};

export const dataAlbums = [
  {
    id: 'xvu-fisherman-sashimi',
    name: 'galleryAlbum',
    body: 'Photos',
    items: [
      {
        weight: 2,
        locale: 'en-US',
        _id: '5cb7b02f70ba7150231feb76',
        creator: 'vancouverdining',
        author: 'x6oc5',
        permlink: 'vancouverdining-5151st60ku3',
        name: 'galleryItem',
        body: 'https://ipfs.busy.org/ipfs/QmYDtPfc4sVpNcwAnNoe5KddWWM3a2TETDz5dyG1JxZhyn',
        id: 'xvu-fisherman-sashimi',
      },
    ],
  },
  {
    id: 'next-id',
    name: 'galleryAlbum_next',
    body: 'Photos',
    items: [
      {
        weight: 2,
        locale: 'en-US',
        _id: '5cb7b02f70ba7150231feb76',
        creator: 'vancouverdining',
        author: 'x6oc5',
        permlink: 'vancouverdining-5151st60ku3',
        name: 'galleryItem',
        body: 'https://ipfs.busy.org/ipfs/QmYDtPfc4sVpNcwAnNoe5KddWWM3a2TETDz5dyG1JxZhyn',
        id: 'xvu-fisherman-sashimi',
      },
    ],
  },
];
