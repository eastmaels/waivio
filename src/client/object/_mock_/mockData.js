export const wObject = {
  _id: '5ce64e338ca3d046336e28c3',
  is_posting_open: true,
  is_extending_open: true,
  weight: 1,
  parent: '',
  children: [],
  author_permlink: 'fti-test3',
  author: 'suy38',
  creator: 'asd09',
  app: 'waiviodev/1.0.0',
  community: '',
  default_name: 'test3',
  object_type: 'test',
  fields: [
    {
      weight: 4,
      locale: 'en-US',
      _id: '5ce64e368ca3d046336e2f65',
      creator: 'asd09',
      author: 'x45ki',
      permlink: 'asd09-kxcburjwxc',
      name: 'name',
      body: 'test3',
      active_votes: [
        {
          _id: '5ce64e398ca3d046336e3608',
          voter: 'asd09',
          weight: 1,
        },
        {
          _id: '5ce64e6c8ca3d046336ec780',
          voter: 'mack-bot',
          weight: -1,
        },
        {
          _id: '5ce64f448ca3d0463371b6a5',
          voter: 'wiv01',
          weight: 1,
        },
        {
          _id: '5ce64f448ca3d0463371b6a6',
          voter: 'x6oc5',
          weight: 1,
        },
        {
          _id: '5ce64f448ca3d0463371b6a8',
          voter: 'sor31',
          weight: 1,
        },
        {
          _id: '5ceee070f3d01e4af3227ae5',
          voter: 'spaminator',
          weight: -1,
        },
      ],
    },
    {
      weight: 1,
      locale: 'en-US',
      _id: '5ce64fe68ca3d0463373cac1',
      creator: 'asd09',
      author: 'q13lp',
      permlink: 'asd09-68lregot8lc',
      name: 'address',
      body:
        '{"accommodation":"ttkiuvuaxsnuukdvdklq","street":"ttkiuvuaxsnuukdvdklq","city":"ttkiuvuaxsnuukdvdklq","country":"ttkiuvuaxsnuukdvdklq"}',
      active_votes: [
        {
          _id: '5ce64fe88ca3d0463373d386',
          voter: 'asd09',
          weight: 1,
        },
        {
          _id: '5ce650198ca3d0463374872b',
          voter: 'mack-bot',
          weight: -1,
        },
        {
          _id: '5ce650398ca3d0463375432e',
          voter: 'an98r',
          weight: 1,
        },
        {
          _id: '5ceee1a6f3d01e4af32a4251',
          voter: 'spaminator',
          weight: -1,
        },
      ],
    },
    {
      weight: 5,
      locale: 'en-US',
      _id: '5ce64ffa8ca3d0463374136f',
      creator: 'asd09',
      author: 'j5gs7',
      permlink: 'asd09-n6cmv8ysec',
      name: 'phone',
      body: 'ttkiuvuaxsnuukdvdklq',
      number: '962675409680',
      active_votes: [
        {
          _id: '5ce64ffd8ca3d04633741d86',
          voter: 'asd09',
          weight: 1,
        },
        {
          _id: '5ce650508ca3d046337599c5',
          voter: 'mack-bot',
          weight: -1,
        },
        {
          _id: '5ce7b294be601c180c57eeda',
          voter: 'w7ngc',
          weight: 1,
        },
        {
          _id: '5ce7b294be601c180c57eedb',
          voter: 'zxc43',
          weight: 1,
        },
        {
          _id: '5ce7b294be601c180c57eedc',
          voter: 'wiv01',
          weight: 1,
        },
        {
          _id: '5ce7b298be601c180c57f0e3',
          voter: 'x6oc5',
          weight: 1,
        },
        {
          _id: '5ceee1baf3d01e4af32aad44',
          voter: 'spaminator',
          weight: -1,
        },
      ],
    },
  ],
  createdAt: '2019-05-23T07:39:31.266Z',
  updatedAt: '2019-05-29T19:47:06.923Z',
  __v: 0,
  preview_gallery: [],
  albums_count: 0,
  photos_count: 0,
  rank: 1,
  followers_count: 2,
  users: [
    {
      weight: 1,
      name: 'asd09',
      rank: 99,
    },
  ],
};

export const obj = {
  _id: '5cdd5e2f915387091ddbf7e6',
  author: 'alexeygrigurko',
  permlink: '31mfpc-title',
  __v: 0,
  abs_rshares: 39228751678,
  active: '2019-05-16T12:57:15',
  active_votes: [
    {
      _id: '5ce40f628ca3d046338ae973',
      voter: 'w7ngc',
      weight: 39229,
      percent: 10000,
      rshares: '39228751678',
    },
  ],
  allow_curation_rewards: true,
  allow_replies: true,
  allow_votes: true,
  app: 'waiviodev/1.0.0',
  author_reputation: 31329420780,
  author_rewards: 0,
  beneficiaries: [
    {
      _id: '5ce40f628ca3d046338ae972',
      account: 'waivio',
      weight: 1000,
    },
  ],
  body:
    'story\n[Bitcoin](http://waiviodev.com/object/ukd-bitcoin)\n[life](http://waiviodev.com/object/life)',
  body_length: 0,
  cashout_time: '2019-05-23T12:57:15',
  category: 'waiviodev',
  children: 0,
  children_abs_rshares: 39228751678,
  created: '2019-05-16T12:57:15',
  createdAt: '2019-05-16T12:57:19.186Z',
  curator_payout_value: '0.000 SBD',
  depth: 0,
  id: 74863195,
  json_metadata:
    '{"community":"waiviodev","app":"waiviodev/1.0.0","format":"markdown","tags":[],"users":[],"links":["http://waiviodev.com/object/ukd-bitcoin","http://waiviodev.com/object/life"],"image":[],"wobj":{"wobjects":[{"objectName":"Bitcoin","author_permlink":"ukd-bitcoin","percent":50},{"objectName":"life","author_permlink":"life","percent":50}]}}',
  last_payout: '1970-01-01T00:00:00',
  last_update: '2019-05-16T12:57:15',
  max_accepted_payout: '1000000.000 SBD',
  max_cashout_time: '1969-12-31T23:59:59',
  net_rshares: 39228751678,
  net_votes: 1,
  parent_author: '',
  parent_permlink: 'waiviodev',
  pending_payout_value: '0.020 SBD',
  percent_steem_dollars: 10000,
  promoted: '0.000 STEEM',
  reblogged_by: [],
  replies: [],
  reward_weight: '10000',
  root_author: 'alexeygrigurko',
  root_permlink: '31mfpc-title',
  root_title: 'title',
  title: 'title',
  total_payout_value: '0.000 SBD',
  total_pending_payout_value: '0.000 STEEM',
  total_vote_weight: 205894,
  updatedAt: '2019-05-21T14:46:58.105Z',
  url: '/waiviodev/@alexeygrigurko/31mfpc-title',
  vote_rshares: 39228751678,
  wobjects: [
    {
      _id: '5cb09ba3b81150400aa6b24a',
      objectName: 'Bitcoin',
      author_permlink: 'ukd-bitcoin',
      percent: 50,
      is_posting_open: true,
      is_extending_open: true,
      weight: 58128.5,
      children: [],
      author: 'sor31',
      creator: 'wiv01',
      app: 'waiviodev/1.0.0',
      community: '',
      default_name: 'bitcoin',
      object_type: 'crypto',
      fields: [
        {
          weight: 2,
          locale: 'en-US',
          _id: '5cb46b1bf03c2c421ef19550',
          creator: 'wiv01',
          author: 'w7ngc',
          permlink: 'cry-bitcoin',
          name: 'name',
          body: 'Bitcoin',
          active_votes: [],
        },
        {
          weight: 1,
          locale: 'en-US',
          _id: '5cb0b3c1f03c2c421e4b3ee5',
          creator: 'wiv01',
          author: 'vmn31',
          permlink: 'tvr-bitcoin-title',
          name: 'title',
          body:
            'Decentralized peer-to-peer electronic cash system without a central bank or single administrator',
          active_votes: [],
        },
        {
          weight: 1,
          locale: 'en-US',
          _id: '5cb0b3c8f03c2c421e4b4f54',
          creator: 'wiv01',
          author: 'w7ngc',
          permlink: 'qql-bitcoin-avatar',
          name: 'avatar',
          body: 'https://ipfs.busy.org/ipfs/QmWDohFQFD3iXnj4ptCRjMk7zSUUHThzaGMHPRESjQRJJK',
          active_votes: [],
        },
        {
          weight: 1,
          locale: 'en-US',
          _id: '5cb0b3caf03c2c421e4b55e3',
          creator: 'wiv01',
          author: 'x6oc5',
          permlink: 'byk-bitcoin-background',
          name: 'background',
          body:
            'https://cdn.steemitimages.com/DQmQWLefDcnYykEq8zSoWfBJyAgi68ZjzNzJwEKWuLT1qzu/Positive%20Vibes%20(6).png',
          active_votes: [],
        },
        {
          weight: 1,
          locale: 'en-US',
          _id: '5cb0b3c4f03c2c421e4b49f0',
          creator: 'wiv01',
          author: 'wiv01',
          permlink: 'crm-bitcoin-description',
          name: 'description',
          body:
            'Cryptocurrency, a form of electronic cash. It is a decentralized digital currency without a central bank or single administrator that can be sent from user-to-user on the peer-to-peer bitcoin network without the need for intermediaries.',
          active_votes: [],
        },
        {
          weight: 1,
          locale: 'en-US',
          _id: '5cb0b3bbf03c2c421e4b32f3',
          creator: 'wiv01',
          author: 'x6oc5',
          permlink: 'kfp-bitcoin-tag',
          name: 'tag',
          body: 'bitcoin',
          active_votes: [],
        },
        {
          weight: 1,
          locale: 'en-US',
          _id: '5cb0b3b7f03c2c421e4b2e2e',
          creator: 'wiv01',
          author: 'w7ngc',
          permlink: 'pnp-bitcoin-chartid',
          name: 'chartid',
          body: 'Bitcoin',
          active_votes: [],
        },
        {
          weight: 1,
          locale: 'en-US',
          _id: '5ce68f92be601c180c46d7cd',
          creator: 'wiv01',
          author: 'no58s',
          permlink: 'icd-bitcoin-filter',
          name: 'newsFilter',
          body:
            '{"allowList":[["bitcoin","crypto"],["bitcoin","cryptocurrancy"],["bitcoin","money"],["bitcoin","blockchain"]],"ignoreList":["photography","food","art","life","busy","blog","kr","haejin","jjm","travel","gaming","dtube","actifit","nature","funnt","steemmonsters","contest","tasteem","drugwars-fight","fight","dw","music","photo"]}',
          active_votes: [],
        },
      ],
      createdAt: '2019-04-12T14:07:31.854Z',
      updatedAt: '2019-05-23T12:18:26.236Z',
      __v: 0,
      parent: '',
      newsFilter: {
        allowList: [
          ['bitcoin', 'crypto'],
          ['bitcoin', 'cryptocurrancy'],
          ['bitcoin', 'money'],
          ['bitcoin', 'blockchain'],
        ],
        ignoreList: [
          'photography',
          'food',
          'art',
          'life',
          'busy',
          'blog',
          'kr',
          'haejin',
          'jjm',
          'travel',
          'gaming',
          'dtube',
          'actifit',
          'nature',
          'funnt',
          'steemmonsters',
          'contest',
          'tasteem',
          'drugwars-fight',
          'fight',
          'dw',
          'music',
          'photo',
        ],
      },
      rank: 1,
    },
    {
      _id: '5caf424c6fb6c810cde66d6b',
      objectName: 'life',
      author_permlink: 'life',
      percent: 50,
      is_posting_open: true,
      is_extending_open: true,
      weight: 32502395738.39817,
      children: [],
      author: 'suy38',
      creator: 'wiv01',
      app: 'waiviodev/1.0.0',
      community: '',
      default_name: 'life',
      object_type: 'hashtag',
      fields: [
        {
          weight: 1,
          locale: 'en-US',
          _id: '5caf424e6fb6c810cde66d6d',
          creator: 'wiv01',
          author: 'x45ki',
          permlink: 'wiv01-iil1j1owkde',
          name: 'name',
          body: 'life',
          active_votes: [],
        },
        {
          weight: 131.20000000000002,
          locale: 'en-US',
          _id: '5cb182f7f03c2c421e8a9afa',
          creator: 'shoppingguide',
          author: 'x6oc5',
          permlink: 'shoppingguide-crdmwrhk36w',
          name: 'avatar',
          body: 'https://ipfs.busy.org/ipfs/QmNokPv2ZTAjXgvukKwGdRDpXqJ2YGPNSyMkVY6GvHRHBs',
          active_votes: [
            {
              _id: '5cb182fbf03c2c421e8a9b80',
              voter: 'shoppingguide',
              weight: 129.20000000000002,
            },
            {
              _id: '5cb424f7f03c2c421ea18890',
              voter: 'jucci',
              weight: 1,
            },
          ],
        },
        {
          weight: 422.70833333333337,
          locale: 'en-US',
          _id: '5cd12114d0555b20e15669a3',
          creator: 'monterey',
          author: 'fgh87',
          permlink: 'monterey-gei9fmdh9w9',
          name: 'tag',
          body: 'someTag',
          active_votes: [
            {
              _id: '5cd12118d0555b20e1567647',
              voter: 'monterey',
              weight: 421.70833333333337,
            },
            {
              _id: '5cd1211cd0555b20e1567c1f',
              voter: 'mack-bot',
              weight: -1,
            },
          ],
        },
      ],
      createdAt: '2019-04-11T13:34:04.222Z',
      updatedAt: '2019-06-17T10:59:49.302Z',
      __v: 0,
      parent: '',
      rank: 8,
    },
  ],
};

export default null;
