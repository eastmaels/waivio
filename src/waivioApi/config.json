{
  "development": {
    "protocol": "https://",
    "": "www.waivio.com",
    "host": "waiviodev.com",
    "": "localhost:8093"
  },
  "staging": {
    "protocol": "https://",
    "host": "waiviodev.com"
  },
  "production": {
    "protocol": "https://",
    "host": "www.waivio.com"
  },
  "apiPrefix": "/api",
  "objectsBotApiPrefix": "/objects-bot",
  "campaignApiPrefix": "/campaigns-api",
  "objectsBot": {
    "createObject": "/create-object",
    "appendObject": "/append-object"
  },
  "createObject": "/create/object",
  "getObjects": "/wobject",
  "searchObjects": "/wobjectSearch",
  "getObjectFollowing": "/following_objects",
  "getObjectFollowers": "/followers",
  "user": "/user",
  "users": "/users",
  "feed": "/feed",
  "posts": "/posts",
  "post": "/post",
  "followingObjects": "/following_objects",
  "getGallery": "/gallery",
  "wobjectsWithUserWeight": "/objects_shares",
  "wobjectsExpertise": "/object_expertise",
  "getObjectTypes": "/objectTypes",
  "objectType": "/objectType",
  "generalSearch": "/generalSearch",
  "userStatistics": "/user-statistics",
  "instrumentStatistic": "/instrument-statistics",
  "campaigns": "/campaigns",
  "suitableUsers": "/suitable_users"
}
