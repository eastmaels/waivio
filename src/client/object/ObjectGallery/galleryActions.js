import _ from 'lodash';
import { createAction } from 'redux-actions';

import { createAsyncActionType } from '../../helpers/stateHelpers';
import { getWobjectGallery } from '../../../waivioApi/ApiClient';

export const GET_ALBUMS = createAsyncActionType('@gallery/GET_ALBUMS');
export const ADD_ALBUM = '@gallery/ADD_ALBUM';
export const ADD_IMAGE = '@gallery/ADD_IMAGE';
export const RESET_GALLERY = '@gallery/RESET_GALLERY';

export const getAlbums = authorPermlink => dispatch =>
  dispatch({
    type: GET_ALBUMS.ACTION,
    payload: getWobjectGallery(authorPermlink),
    meta: { authorPermlink },
  });

export const addAlbumToStore = createAction(ADD_ALBUM);
export const addImageToAlbumStore = createAction(ADD_IMAGE);
export const resetGallery = createAction(RESET_GALLERY);
