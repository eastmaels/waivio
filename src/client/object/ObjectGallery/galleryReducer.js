import _ from 'lodash';
import * as galleryActions from './galleryActions';

const defaultState = {
  albumsLoading: true,
  error: null,
  albums: [],
};

export const mapAlbums = (albums, authorPermlink) => {
  const defaultAlbum = _.remove(albums, alb => alb.id === authorPermlink);
  const sortedAlbums = _.orderBy(albums, ['weight'], ['desc']);

  return [...defaultAlbum, ...sortedAlbums];
};

export default (state = defaultState, action) => {
  switch (action.type) {
    case galleryActions.GET_ALBUMS.START:
      return {
        ...state,
        error: null,
      };
    case galleryActions.GET_ALBUMS.ERROR:
      return {
        ...state,
        error: action.message,
        albumsLoading: false,
      };
    case galleryActions.GET_ALBUMS.SUCCESS:
      return {
        ...state,
        albums: mapAlbums(action.payload, action.meta.authorPermlink),
        error: null,
        albumsLoading: false,
      };
    case galleryActions.ADD_ALBUM: {
      return {
        ...state,
        albums: [...state.albums, action.payload],
      };
    }
    case galleryActions.ADD_IMAGE: {
      const albums = state.albums.map(album =>
        album.id === action.payload.id
          ? {
              ...album,
              items: [...album.items, action.payload],
            }
          : album,
      );
      return { ...state, albums };
    }
    case galleryActions.RESET_GALLERY: {
      return defaultState;
    }
    default:
      return state;
  }
};

export const getObjectAlbums = state => state.albums;
export const getIsObjectAlbumsLoading = state => state.albumsLoading;
