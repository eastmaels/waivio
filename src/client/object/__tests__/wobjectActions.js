import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import * as actions from '../wobjectsActions';
import { getObject, getUsersByObject, getFeedContentByObject } from '../../../waivioApi/ApiClient';
import { createPermlink } from '../../vendor/steemitHelpers';
import { getAlbums } from '../../object/ObjectGallery/galleryActions';
import { wObject, obj } from '../_mock_/mockData';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

jest.mock('../../../waivioApi/ApiClient');
jest.mock('../../object/ObjectGallery/galleryActions');
jest.mock('../../vendor/steemitHelpers');

describe('object actions', () => {
  it('should dispatch actions of GET_OBJECT_SUCCESS', () => {
    const expectedActions = [
      {
        type: actions.GET_OBJECT_SUCCESS,
        payload: {},
      },
    ];

    const store = mockStore({});

    store.dispatch(actions.clearObjectFromStore());
    expect(store.getActions()).toEqual(expectedActions);
  });

  it('should dispatch actions of ADD_ITEM_TO_LIST', () => {
    const expectedActions = [
      {
        type: actions.ADD_ITEM_TO_LIST,
        payload: 'Anna123',
      },
    ];

    const store = mockStore({});

    store.dispatch(actions.addListItem('Anna123'));
    expect(store.getActions()).toEqual(expectedActions);
  });

  it('should dispatch actions of GET_OBJECT', () => {
    getObject.mockImplementation(() => wObject);
    const expectedActions = [
      {
        type: actions.GET_OBJECT,
        payload: wObject,
      },
    ];

    const store = mockStore({});

    store.dispatch(actions.getObject('randomValue', 'Anna123'));
    expect(store.getActions()).toEqual(expectedActions);
  });

  it('should dispatch actions of GET_USERS_BY_OBJECT', () => {
    getUsersByObject.mockImplementation(() => wObject);
    const expectedActions = [
      {
        type: actions.GET_USERS_BY_OBJECT.ACTION,
        payload: wObject,
      },
    ];

    const store = mockStore({});

    store.dispatch(actions.getUsersByObject('fti-test3'));
    expect(store.getActions()).toEqual(expectedActions);
  });

  it('should dispatch actions of GET_FEED_CONTENT_BY_OBJECT', () => {
    getFeedContentByObject.mockImplementation(() => obj);
    const expectedActions = [
      {
        type: actions.GET_FEED_CONTENT_BY_OBJECT.ACTION,
        payload: obj,
      },
    ];

    const store = mockStore({});

    store.dispatch(actions.getFeedContentByObject('fti-test3'));
    expect(store.getActions()).toEqual(expectedActions);
  });

  // it('should dispatch actions of getObjectInfo', () => {
  //   const store = mockStore({});
  //   store.dispatch(actions.getObjectInfo('randomValue', 'fti-test3'));
  //   expect(store.getActions()).toEqual('a');
  // });
  //
  // it('should dispatch actions of getObjectInfo', () => {
  //   createPermlink.mockImplementation(() => {});
  //   const postData = {
  //     field: {
  //       locale: 'en-US',
  //       name: 'galleryAlbum',
  //       body: 'galleryAlbumForm',
  //       id: 'randomIdValue',
  //     },
  //     author: 'Ivan',
  //     permlink: 'randomLinkValue',
  //   };
  //
  //   const store = mockStore({});
  //   store.dispatch(actions.createWaivioObject(postData));
  //   expect(store.getActions()).toMatchSnapshot();
  // });
});
