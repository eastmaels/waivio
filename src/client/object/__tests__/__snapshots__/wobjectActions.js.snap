// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`object actions should dispatch actions of GET_USERS_BY_OBJECT 1`] = `[Function]`;

exports[`object actions should dispatch actions of GET_USERS_BY_OBJECT: 1 1`] = `[Function]`;

exports[`object actions should dispatch actions of getObjectInfo 1`] = `[Function]`;

exports[`object actions should dispatch actions of getObjectInfo 2`] = `[Function]`;

exports[`object actions should dispatch actions of getObjectInfo: 1 1`] = `[Function]`;

exports[`object actions should dispatch actions of getObjectInfo: 1 2`] = `[Function]`;
